roles/ceph-osd/tasks/system_tuning.yml

```diff
@@ -0,0 +1,28 @@
+---
+- name: disable osd directory parsing by updatedb
+  command: updatedb -e /var/lib/ceph
+  changed_when: false
+  failed_when: false
+
+- name: disable transparent hugepage
+  lineinfile: dest=/etc/default/grub
+    state=present
+    regexp="^GRUB_CMDLINE_LINUX_DEFAULT=\"(.*)\"$"
+    line="GRUB_CMDLINE_LINUX_DEFAULT=\"\1 transparent_hugepage=never\""
+    backrefs=yes
+  when: ceph_osd.disable_transparent_hugepage
```

Comment:
Just as a knowledge bit, this setting won't take effect until a reboot. We don't normally reboot. Is there a way to make this happen live, as well as on next reboot?